cmd/tools/generate/counter.go

```diff
@@ -726,6 +779,10 @@ func generateCounterTemplate(counters map[string]Counter, version [3]int) {
 		panic(err)
 	}
 	fmt.Printf("Harvest metric documentation generated at %s \n", targetPath)
+
+	if issues {
```

Comment:
Instead of tracking through a boolean, you could check if the table has any rows
`if table.NumLines() > 0`